docs/administering/sample-config/nginx-example.conf

```diff
@@ -24,7 +24,7 @@ server {
 # Configuration for Sandstorm shell and apps, over HTTPS.
 server {
   listen 443;
-  server_name example.com *.example.com;
+  server_name example.com *.example.com ~^sandstorm-[^.]*\.example\.com$;
```

Comment:
My reading of this is that `~^sandstorm-[^.]*\.example\.com$` should already be covered by `*.example.com`.  Am I missing something?